frontend/src/component/common/Markdown/Markdown.tsx

```diff
@@ -0,0 +1,24 @@
+import { Link } from '@mui/material';
+import { AnchorHTMLAttributes, ComponentProps } from 'react';
+import ReactMarkdown from 'react-markdown';
+import { useNavigate } from 'react-router-dom';
+
+const LinkRenderer = ({
+    href = '',
+    children,
+}: AnchorHTMLAttributes<HTMLAnchorElement>) => {
+    const navigate = useNavigate();
+
+    if (href.startsWith('/'))
```

Comment:
Relative links can also start without the slash, maybe just:
```suggestion
    if (!href.match(/^https?:\/\//))
```

Reply:
Hmm true, but the same can be said about absolute links. `mailto:` should not open a new tab, even though it doesn't start with `http*`. I'm somewhat confident with this for now, since it's the same logic that we're following for the banner link. If we decide to change it, we should change it in both places.

Reply:
More context: https://github.com/Unleash/unleash/pull/5011

Reply:
Sure, it makes sense. Feel free to merge it as is but I think we can potentially support other protocols with the regexp as well. This is what I like the most, being explicit about which protocols will open in a new tab:
```typescript
if (href.match(/^(https?:\/\/|mailto:)/))
    return (
        <Link href={href} target='_blank' rel='noreferrer'>
            {children}
        </Link>
    );
return (
        <Link href={href} rel='noreferrer'>{children}</Link>
);
```

Reply:
In general we know about three scenarios we actually use:

1. Internal links. We want these to not open In a new tab. 
2. External links. We want these to open in a new tab. 
3. Mailto links. We want these to open in a new tab. This is particularly useful if the use has configured Gmail as the default email client. 

The current solution from @nunogois solves all these use cases In a very simple and easy to understand way. I see no reason for complicated regex to support possible scenarios in the future.

Reply:
Yes, I can see this handling absolute internal URLs, but if this can be an input from a user they could also be: `href="feature/variants"`. We could also validate and require absolute internal URLs, or just consider this an edge case we don't want to cover (i.e. internal links that are not absolute will open in a new tab). 

In any case, I'm not considering this a blocker for the PR